### PR TITLE
Drop media link from twitter text

### PR DESCRIPTION
**Pull request by raboof**
I'm open to recommendations on making the tests nicer ;)

### CODE REVIEW

**Comment by yogthos** [comment]
Hi, thanks and it might be better to put the test data in `.edn` files in the test directory. You can read them in with `(-> "path/to/file" (fs/readFileSync #js {:encoding "UTF-8"}) edn/read-string)`.

File: mastodon_bot/core.cljs [hunk 1]
```diff
@@ -138,12 +138,20 @@
       (when-not (:media-only? mastodon-config)
         (post-status text)))))
 
+(defn in [needle haystack]
+  (some (partial = needle) haystack))
+
+; If the text ends in a link to the media (which is uploaded anyway),
+; chop it off instead of including the link in the toot
+(defn chop-tail-media-url [text media]
+  (clojure.string/replace text #" (\S+)$" #(if (in (%1 1) (map :url media)) "" (%1 0))))
```
Comment (yogthos) [inline mastodon_bot/core.cljs]: This could just be `(string/replace ...)` since `clojure.string` is already aliased in the namespace.

**Comment by raboof** [comment]
thanks, updated

**Comment by yogthos** [comment]
👍

**Pull request merged**